flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -261,6 +248,14 @@ TestSpec withFunction(Class<? extends UserDefinedFunction> functionClass) {
 
         TestSpec testTableApiResult(
                 Expression expression, Object result, AbstractDataType<?> dataType) {
+            return testTableApiResult(
+                    new Expression[] {expression},
+                    new Object[] {result},
+                    new AbstractDataType<?>[] {dataType});
+        }
+
+        TestSpec testTableApiResult(
+                Expression[] expression, Object[] result, AbstractDataType<?>[] dataType) {
```

Comment:
nit: could we use `List` instead of arrays? I find `Arrays.asList` and soon `List.of` nicer than the verbose array syntax.

Reply:
Don't have any objection, just chose arrays here to avoid the overhead of List objects especially for the common cases of only one element, but considering it's test, Lists are nicer indeed.